components/Link.js

```diff
@@ -9,6 +9,7 @@ export default function Link({ children, className, rel, ...restProps }) {
           ? className
           : "text-blue-600 underline decoration-dotted dark:text-blue-500 hover:underline hover:decoration-solid"
       }
+      prefetch={false}
```

Comment:
Do we want to have a condition so it only happens on mobile?

Reply:
I tried that, but webpack still bundles the code to preload the JavaScript files if the preload is set by a conditional. It is around 700k of JS that gets loaded by the main-XXX chunk of the bundle without preload set to false.

Reply:
ouch, ok thank you for clarifying 👍